src/qt/ApplicationHandler.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <QtCore/QObject>
+#include <QtCore/QTranslator>
+#include <QtQml/qqmlregistration.h>
+
+class ApplicationHandler : public QObject
+{
+   Q_OBJECT
+   QML_ELEMENT
+
+public:
+   explicit ApplicationHandler(QObject *parent = nullptr);
+   virtual ~ApplicationHandler() = default;
+
+   Q_INVOKABLE void ChangeToEnglish();
```

Comment:
Ideally it should be `ChangeLanguage(const QString& code)`

Reply:
Done